packages/mask/package.json

```diff
@@ -29,7 +29,10 @@
     }
   },
   "exports": {
-    ".": null
+      "./crypto": {
+          "mask-src": "./background/services/crypto/index.ts",
+          "default": "./background/services/crypto/index.js"
+      }
```

Comment:
this package should not have any exports. can you explain why you need this?

Reply:
We would decode encrypted post payload in firefly.

Reply:
you should use `@masknet/encryption` package directly, which is published on npm. you can see how `packages/app` uses that package as a reference.